jupyterlab_gcpscheduler/package.json

```diff
@@ -52,7 +52,7 @@
     "@phosphor/signaling": "^1.3.0",
     "csstips": "^1.2.0",
     "formik": "^1.5.8",
-    "gcp_jupyterlab_shared": "1.1.2",
+    "gcp_jupyterlab_shared": ">=1.1.2",
```

Comment:
I think using caret `^1.1.2` is best here since it will ensure we remain on the same major version.

Reply:
Ah okay. I thought since 1.1.2 had a breaking change (more icons) it had to be >=. Changed!